pkgs/applications/misc/koreader/vendored.nix

```diff
@@ -0,0 +1,233 @@
+# The bad news: the contents of this file are updated by hand because
+# the upstream data are in "cmake syntax" for which there does not
```

Comment:
Do you believe that even a Perl quick&dirty script would be harder and error-prone? Just out of curiousity.

Reply:
Yes I believe it would be error-prone.  I'm also not a fan of "parsing" with regexes. 

I volunteer to manually maintain `vendored.nix`.